### PR TITLE
Feature: Allow bars in discreteBar chart to have rounded corners

### PR DESCRIPTION
**Pull request by avdeveloper**

### CODE REVIEW

File: src/models/discreteBar.js [hunk 1]
```diff
@@ -220,6 +222,7 @@ nv.models.discreteBar = function() {
         height:  {get: function(){return height;}, set: function(_){height=_;}},
         forceY:  {get: function(){return forceY;}, set: function(_){forceY=_;}},
         showValues: {get: function(){return showValues;}, set: function(_){showValues=_;}},
+        rx:      {get: function(){return rx;}, set: function(_){rx=_}},
```
Comment (liquidpele) [inline src/models/discreteBar.js]: "rx" isn't very descriptive, I'd like to name it something more like "cornerRadius" so it's clear without reading the code what it does.

**Comment by liquidpele** [comment]
for new options, please also add a description/example in the json object in the bottom of documentation.html

**Comment by avdeveloper** [comment]
@liquidpele done. ready for merge

**Comment by liquidpele** [comment]
Thanks for contributing!

**Comment by zuzusik** [comment]
Any idea if this is going to be released to NPM anytime soon?

**Comment by liquidpele** [comment]
I'm not involved with novus anymore sorry...  but I am managing the http://github.com/nvd3/nvd3 project now which I'd like to be a D3v4 port.  If you're interested in helping please hop on over :)

**Pull request merged**